### DISCUSSION TITLE
SSL: error:0200100D:system library:fopen:Permission denied:fopen('/home/liuph/kong/bazel-bin/build/kong-dev/kong/servroot/.ca_combined','r')

### DISCUSSION THREAD

**Discussion by liupenghui-owen** [Kong/kong#10503] [Help]
Hi, during Kong start, I found below error, is there any quick guidance for this problem? thanks ! :)

2023/03/16 10:29:43 [emerg] 229298#0: SSL_CTX_load_verify_locations("/home/kong/bazel-bin/build/kong-dev/kong/servroot/.ca_combined") failed (SSL: error:0200100D:system library:fopen:Permission denied:fopen('/home/liuph/kong/bazel-bin/build/kong-dev/kong/servroot/.ca_combined','r') error:2006D002:BIO routines:BIO_new_file:system lib error:0B084002:x509 certificate routines:X509_load_cert_crl_file:system lib)

**Comment by ghost** [comment]
The error message indicates that there is a permission issue when loading the certificate authority file for SSL and the certificate cannot be loaded due to a permission denied error. Here are the steps to resolve this issue:

1. Check if the file permissions are correct. The certificate file should have read permissions. Use the “chmod” command to modify permissions: `chmod 644 .ca_combined`.
2. Check if the file ownership is correct. The certificate file should belong to the user who runs Kong.
